docs/src/main/sphinx/release/release-394.md

```diff
@@ -0,0 +1,39 @@
+# Release 394 (24 August 2022)
+
+## General
+
+* Improve performance of joins which use a single `bigint` column. ({issue}`13432`)
+* Ensure `UPDATE` queries cannot insert null values into columns which are
+  `NOT NULL`. ({issue}`13435`)
+* Fix failure when an `UPDATE` query uses a `WHERE` clause which always
+  evaluates to false. ({issue}`12422`)
+* Fix potential failure for queries involving joins and implicit or explicit
+  casts of `null` to a concrete type. ({issue}`13565`)
+
+## Docker image
+
+* Terminate the JVM if it is unable to create a native thread. ({issue}`13736`)
```

Comment:
This needs to be rephrased in a more user-friendly way. Users don't know or care about when a native thread can't be created.

Reply:
@electrum any recommendations?

Reply:
Maybe 

* Ensure Trino stops running with insufficient resources to avoid partial failures.